class.jetpack-modules-list-table.php

```diff
@@ -50,6 +50,7 @@ function __construct() {
 			'i18n'    => array(
 				'search_placeholder' => __( 'Search Modules…', 'jetpack' ),
 			),
+			'modalinfo' =>  isset( $_GET['info'] ) ? $_GET['info'] : false,
```

Comment:
We should probably do some sanitization before passing this over.  Currently if I write something like `&info=flippideedoo` it'll pop a blank modal.